_posts/2017-11-20-cfst-open.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: Call for Specialist Tracks is now open!
+---
+
+
+Each year at PyCon AU, we dedicate a day to our Specialist Tracks. These are single-day “mini-conferences”, where each room's talks have a specific theme, curated by members of the community.
+
+
+In the past we have had tracks for Science and Data, Education, Internet of Things, Django, OpenStack, and more. If are a fan of any of the previous topics and would like to run the Track in 2018, we want to hear from you! If you have something new you want to try out, we want to hear from you!
```

Comment:
Missed a word here. "If you are a fan..."

Reply:
... after fixing that, :shipit: